src/storage/heap_file.c

```diff
@@ -26461,3 +26246,189 @@ heap_update_set_prev_version (THREAD_ENTRY * thread_p, const OID * oid, PGBUF_WA
 end:
   return error_code;
 }
+
+/*
+ * heap_get_internal2 () - Generic function for retrieving last version of heap objects, not visible versions
+ * 
+ *
+ */
```

Comment:
@andrei14vl 

Would you update/add the function comments for the new functions?